charts/sonarqube/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [6.1.2]
+* Refactor Pdb and Ingress to be compatible with static compatibitly test
```

Comment:
we drop some apiVersions here as well. Don't we need to say that we set a minimum compatibility?

Reply:
The minimum compatibility is the one defined by the chart itself, resolved, see ongoing discussion [here](https://github.com/SonarSource/helm-chart-sonarqube/pull/270#discussion_r1009176505)